### PR TITLE
openrc: increase default ulimit

### PR DESCRIPTION
**Pull request by wenerme**
## Description

```bash
 sudo head -1000 /var/log/k0s.log | grep 'too many open files'| wc -l
```

```
252
```

fix konnectivity-server.sock: socket: too many open files

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## How Has This Been Tested?

- [x] Manual test
- [ ] Auto test added

## Checklist:

- [x] My code follows the style [guidelines](https://github.com/k0sproject/k0s/blob/main/docs/contributors/overview.md) of this project 
- [x] My commit messages are [signed-off](https://github.com/k0sproject/k0s/blob/main/docs/contributors/github_workflow.md)
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by wenerme** [comment]
@twz123 alpine is default to 1024/4096, this ulimit is copy from k3s https://gitlab.alpinelinux.org/alpine/aports/-/blob/master/community/k3s/k3s.initd#L20

**Comment by jnummelin** [comment]
@ncopa as Alpine wizard, PTAL 😄

File: pkg/install/linux_openrc.go [hunk 1]
```diff
@@ -28,6 +28,8 @@ command_args="{{range .Arguments}}{{.}} {{end}}"
 name=$(basename $(readlink -f $command))
 supervise_daemon_args="--stdout /var/log/${name}.log --stderr /var/log/${name}.err"
 
+rc_ulimit="${K0S_ULIMIT:--c unlimited -n 1048576 -u unlimited}"
```
Comment (ncopa) [inline pkg/install/linux_openrc.go:31]:
```suggestion
: "${rc_ulimit=-c unlimited -n 1048576 -u unlimited}"
```

This way user can override the setting in `/etc/conf.d/k0sworker` using `rc_ulimit`. It is also backwards compatible for users who already may have set `rc_ulimit` in their `/etc/conf.d/k0sworker`

**Review by ncopa — CHANGES_REQUESTED** [review]
would be nice to be backwards compatible for users who may have already set `rc_ulimit` in their conf.d file.

File: pkg/install/linux_openrc.go (hunk 1 above)
Comment (ncopa) [reply pkg/install/linux_openrc.go:31]: actually, I don't think we should set `-c unlimited` by default for production.

Comment (wenerme) [reply pkg/install/linux_openrc.go]: core file size default to zero, but leave -c untouched is ok for me.

**Review by ncopa — APPROVED** [review]

**Pull request merged**